docs/walkthrough.md

```diff
@@ -32,17 +32,21 @@ be done with Helm setup.
 If you don't already have Helm v2, see the
 [installation instructions](https://github.com/kubernetes/helm/blob/master/docs/install.md).
 
-### RBAC Considerations
-
-If your kubernetes cluster has [RBAC](https://kubernetes.io/docs/admin/authorization/rbac/)
-enabled, you must ensure that the default service account for the `kube-system`
-namespace has the `cluster-admin` role:
+If your kubernetes cluster has
+[RBAC](https://kubernetes.io/docs/admin/authorization/rbac/) enabled, you must
+ensure that tiller pod has `cluster-admin` access. The default install of tiller
+done with `helm init` installs the tiller pod into `kube-system` namespace and
```

Comment:
I think grammatically it's fine, but @nilebox 's rephrasing is more clear so I'm also in favor of the change. Though I'd say no comma after "namespace".

@MHBauer I'd also think you dropped a "the" on line 37 "... ensure that tiller pod..."